cosmwasm/contracts/wattpeak-minter/src/query.rs

```diff
@@ -179,18 +200,26 @@ mod tests {
                 description: "test description".to_string(),
                 document_deal_link: "ipfs://test-link".to_string(),
                 max_wattpeak: 1000,
+                image_link: "ipfs://test-image".to_string(),
+                location: Location {
+                    latitude: Decimal::from_ratio(1001u64, 10u64),
+                    longitude: Decimal::from_ratio(1001u64, 10u64),
+                },
             };
             execute(deps.as_mut(), mock_env(), info.clone(), upload_first).unwrap();
 
             let res = query(deps.as_ref(), mock_env(), crate::msg::QueryMsg::Project {
                 id: 1,
             }).unwrap();
+            
             let project: crate::state::Project = from_json(&res).unwrap();
             assert_eq!(project.name, "test name");
             assert_eq!(project.description, "test description");
             assert_eq!(project.document_deal_link, "ipfs://test-link");
             assert_eq!(project.max_wattpeak, 1000);
             assert_eq!(project.minted_wattpeak_count, 0);
+            assert_eq!(project.image_link, "ipfs://test-image");
+            assert_eq!(project.location.latitude, Decimal::from_ratio(1001u64, 10u64));
```

Comment:
Check the longitude as well.

Could you also do a verification of what the string representation would look like for both latitude and longitude, so we are sure it comes out as expected to the client?

Reply:
Sorted!